source/hic_hal/stm32/stm32h743ii/extended_features/include/adapter.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _ADAPTER_H_INCLUDED_
+#define _ADAPTER_H_INCLUDED_
+
+typedef enum
+{
+    ADAPTER_UDB_13_FLEX,
+    ADAPTER_JOKER_FLEX,
+    ADAPTER_UDB_6_FLEX,
+    ADAPTER_UDB_17_FLEX,
+    ADAPTER_UDB_12_FLEX,
+    ADAPTER_UNKNOWN
+} adapter_type_t;
+
+#define ADAPTER_UDB_13_FLEX_ADC_MIN     (54000)
+#define ADAPTER_JOKER_FLEX_ADC_MIN      (51000)
+#define ADAPTER_UDB_6_FLEX_ADC_MIN      (47000)
+#define ADAPTER_UDB_17_FLEX_ADC_MIN     (44000)
+#define ADAPTER_UDB_12_FLEX_ADC_MIN     (40000)
```

Comment:
Are there already reserved values for future adapters? Can we go ahead and set them up here ahead of time so that we can return a correct adapter_type_t even if we don't know which specific adapter it is yet?

Reply:
I have no idea if there are reserved value. But I think it is hard to name those adapters, I will just use ADAPTER_UNKNOWN now.